src/agent/transform-headers-agent.ts

```diff
@@ -12,11 +13,11 @@ class TransformHeadersAgent extends WrappedAgent {
     /**
      * @description Transforms the request via header normalization.
      * @see {TransformHeadersAgent.toPascalCase}
-     * @param {http.ClientRequest} request
+     * @param {ClientRequest} request
      * @param {string[]} sortedHeaders - headers in order, optional
```

Comment:
I think we can delete the `@param` annotations when we have TS now. The comments might get out of sync with the actual types over time.